syntax = "proto3";

message ChatRequest {
    string action = 1;
    string value = 2;
}

message ChatMessage {
    string type = 1;
    string userName = 2;
    string message = 3;
}